packages/lwc-services/src/utils/webpack/webpack-builder.ts

```diff
@@ -109,6 +104,29 @@ function buildWebpackConfig({ entries, outputDir, moduleDir, mode }: string) {
             }
         }
     }
+
+    if (customConfig) {
+        serverConfig = webpackMerge.smart(serverConfig, customConfig)
+    }
+
+    const lwc_module_resolver = {
+        resolve: {
+            alias: {
+                lwc: require.resolve('@lwc/engine'),
+                '@lwc/wire-service': require.resolve('@lwc/wire-service')
+            },
+            plugins: [
+                new ModuleResolver({
+                    module: MODULE_CONFIG,
+                    entries: Object.keys(serverConfig.entry).map(
+                        key => serverConfig.entry[key]
+                    )
+                })
+            ]
+        }
+    }
+    serverConfig = webpackMerge.smart(serverConfig, lwc_module_resolver)
```

Comment:
This `merge` doesn't compile because of the [format requirements](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/webpack/index.d.ts#L50).